src/XenCert/StorageHandler.py

```diff
@@ -1847,7 +1847,8 @@ def __init__(self, storage_handler):
     def Gfs2Supported(self):
         pools = self.session.xenapi.pool.get_all_records()
         pool = [pools[k] for k in pools][0]
-        if not 'restrict_corosync' in pool['restrictions']:
+        if not 'restrict_corosync' in pool['restrictions'] or \
+                self.storage_conf['storage_type'] == "fcoe":
```

Comment:
@MarkSymsCtx Additional check

Reply:
Does it indicate GFS2oFCoE is not supported?

Reply:
Correct, GFS2 only supports iSCSI and FC not FCoE

Reply:
Got it.